### PR TITLE
Clip examples with pythonic interactions.

### PR DESCRIPTION
**Pull request by gsaluja9**
Add Clips as a Subclass of Entities, and updates the example to add videos with clips using DataModels.

### CODE REVIEW

**Review by vishakha041 — APPROVED** [review]
Some simple suggestions for our repo.

File: aperturedb/Query.py [hunk 1]
```diff
@@ -334,7 +334,10 @@ def spec(cls,
              sort: Sort = None,
              list: List[str] = None,
              group_by_src: bool = False,
-             blobs: bool = False
+             blobs: bool = False,
+             set: str = None,
```
[the remark this answers is not in the document]
Comment (gsaluja9) [reply aperturedb/Query.py:338]: (https://github.com/aperture-data/aperturedb-python/issues/490)

File: aperturedb/Query.py [hunk 2]
```diff
@@ -357,7 +360,10 @@ def spec(cls,
             sort = sort,
             list = list,
             blobs=blobs,
-            group_by_src = group_by_src
+            group_by_src = group_by_src,
+            set=set,
+            vector=vector,
```
[the remark this answers is not in the document]
Comment (gsaluja9) [reply aperturedb/Query.py:365]: Very basic level. and not sure about the comprehensiveness.

File: examples/loading_with_models/add_video_model.py [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
-from typing import List
+import json
 
-# Define the models for the associstaion of Video, Video Clips, and Embeddings
+# In aperturedb we have Videos, Video Clips and Embeddings(aka Descriptors)
+#recognized as first class objects.
 # Note : Video has multiple Clips, and each Clip has an embedding.
 
+# In aperturedb.datamodel, we already define datamodels for Videos, Clips and Embeddings
+# Now Define the data models for the "association" of Video, Video Clips, and Embeddings
```
[the remark this answers is not in the document]
Comment (gsaluja9) [reply examples/loading_with_models/add_video_model.py:13]: Oh, the nomenclature was to suggest what this script does. Like Add a Video (to the DB) based on a DataModel. This was to extend our Entity example, where we associate  the hand , finger and such.

File: examples/loading_with_models/get_tl_embeddings.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            'start_offset_sec': v.start_offset_sec,
+            'end_offset_sec': v.end_offset_sec,
+            'embedding_scope': v.embedding_scope
+        })
+
+    return embeddings, task_result
+
+
+def generate_text_embeddings(text: str):
+    text_embedding = twelvelabs_client.embed.create(
```
[the remark this answers is not in the document]
Comment (gsaluja9) [reply examples/loading_with_models/get_tl_embeddings.py:46]: yea, the snippets here were used to indicate how everything has been obtained. These examples do not get packaged or define requirements though. They serve as a building blocks to explain a concept. We can omit it.

**Comment by gsaluja9** [comment]
Feel free to review this later. Merging this to make a release.

**Pull request merged**